log.go

```diff
@@ -0,0 +1,19 @@
+package trails
+
+import "golang.org/x/exp/slog"
+
+const (
+	LogKindKey = "kind"
```

Comment:
you prefer the germanic "kin" over the greek/latin "type" eh? 😄

Reply:
I meant this as a dumb joke, but now that I see `slog` has a `Kind` type I imagine that's where you might've been going with this 👍

Reply:
Haha, nice. It's more that `type` is a reserved keyword so don't want any shenanigans running into it when naming fields, vars, etc.